sdks/typescript/featurehub-javascript-client-sdk/app/feature_hub_config.ts

```diff
@@ -36,10 +36,6 @@ export class FHLog {
     FHLog.fhLog.trace = (...args: any[]) => {
     };
   }
-
-  public Замолчи() {
```

Comment:
we had people using this which is why it is left in, by removing it you are breaking their API.

Reply:
i.e. please deprecate so we can remove in a future version.